components/SideDrawer.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+    Drawer,
+    DrawerBody,
+    DrawerFooter,
+    DrawerHeader,
+    DrawerOverlay,
+    DrawerContent,
+    DrawerCloseButton,
+    Button,
+    useDisclosure,
+    Input,
+    IconButton,
+    Flex,
+    Box,
+    Text
+
+} from '@chakra-ui/react'
+import React from 'react'
+
+import { useRef } from 'react'
+import HamburgerIcon from './HamburgerIcon'
+
+
+function SideDrawer() {
+    const { isOpen, onOpen, onClose } = useDisclosure()
+
+    return (
+        <Flex justify={'end'}>
```

Comment:
We can remove this element since we're wrapping it in a `Flex` in `pages/index.tsx`